Configurations/10-main.conf

```diff
@@ -683,6 +683,7 @@ my %targets = (
         CXXFLAGS         => picker(default => "-Wall",
                                    debug   => "-O0 -g",
                                    release => "-O3"),
+        LDFLAGS          => "-Wl,--allow-multiple-definition",
```

Comment:
This change only provides a workaround for multiple definitions within the OpenSSL project.  However, libcrypto.a still contains these definitions, which may cause issues for software that uses these functions and links with the static library.
I suggest adding a macro, such as -DOPENSSL_SHLIB=1, to enable the symbol versioning thing exclusively for the shared library.  I believe his shouldn't be a problem since it looks like OpenSSL already compiles two sets of object files separately for static and dynamic libraries.

Reply:
Yes, it's a reasonable (and I think, inevitable) way forward